src/Tests/dotnet-add-package.Tests/GivenDotnetPackageAdd.cs

```diff
@@ -41,6 +41,61 @@ public void WhenValidPackageIsPassedBeforeVersionItGetsAdded()
             cmd.StdErr.Should().BeEmpty();
         }
 
+        public static readonly List<object[]> AddPkg_PackageVersionsLatestPrereleaseSucessData
+            = new List<object[]>
+            {
+                    new object[] { new string[] { "0.0.5", "0.9.0", "1.0.0-preview.3" }, "1.0.0-preview.3" },
+                    new object[] { new string[] { "0.0.5", "0.9.0", "1.0.0-preview.3", "1.1.1-preview.7" }, "1.1.1-preview.7" },
+                    new object[] { new string[] { "0.0.5", "0.9.0", "1.0.0" }, "1.0.0" },
+                    new object[] { new string[] { "0.0.5", "0.9.0", "1.0.0-preview.3", "2.0.0" }, "2.0.0" },
+                    new object[] { new string[] { "1.0.0-preview.1", "1.0.0-preview.2", "1.0.0-preview.3" }, "1.0.0-preview.3" },
+            };
+
+        [Theory]
+        [MemberData(nameof(AddPkg_PackageVersionsLatestPrereleaseSucessData))]
+        public void WhenPrereleaseOptionIsPassed(string[] inputVersions, string expectedVersion)
+        {
+            var targetFramework = "netcoreapp5.0";
+            TestProject testProject = new TestProject()
+            {
+                Name = "Project",
+                IsSdkProject = true,
+                IsExe = false,
+                TargetFrameworks = targetFramework,
+            };
+
+            var packages = inputVersions.Select(e => GetPackagePath(targetFramework, "A", e)).ToArray(); 
+
+            testProject.AdditionalProperties.Add("RestoreSources",
+                                     "$(RestoreSources);" + Path.GetDirectoryName(packages[0]));
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+
+            var buildCommand = new BuildCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name));
+
+            var cmd = new DotnetCommand(Log)
+                .WithWorkingDirectory(buildCommand.ProjectRootPath)
```

Comment:
It looks like this is the only place this build command is used, can we pass the path directly and delete the build command?